pkg/cli/admin/catalog/build.go

```diff
@@ -26,13 +27,29 @@ var (
 
 		Extracts the contents of a collection of operator manifests to disk, and builds them into
 		an operator registry catalog image.
+
+		The base image used for the catalog should match the target version of ocp. This can be set manually with 
+		the '--from' flag, or it can be inferred from an ocp release image via '--release-image', or directly from
+		the target cluster if logged in via 'oc login'. 
+
+		The base image will often be a multi-arch image. By default, the linux/amd64 variant is chosen. This can be 
+		overridden with '--filter-by-os'.
 		`)
 	buildExample = templates.Examples(`
-# Build an operator catalog from an appregistry repo and store in a file 
+# Build an operator catalog from an appregistry repo and store in a file. Assumes you are logged in via 'oc login'.
```

Comment:
Every command in oc assumes you're logged in, so I don't think you need to state that here explicitly.